api/src/main/java/de/adorsys/keymanagement/api/keystore/KeyStoreView.java

```diff
@@ -35,7 +35,23 @@
      */
     <Q> AliasView<Q> allAliases();
 
+    /**
+     * Get source of keys
```

Comment:
Get source of keys. For example KeySource can wrap following entities - java Keystore, Database table, list of Key instances, etc.

Reply:
done